stograde/toolkit/__main__.py

```diff
@@ -155,8 +156,11 @@ def main():
                 else:
                     sys.exit(1)
 
-    if re_cache_specs and os.path.exists("data/_cache"):
-        os.removedirs("data/_cache")
+    if re_cache_specs:
+        try:
+            shutil.rmtree("data/specs/_cache")
+        except OSError:
+            pass
 
     specs = load_all_specs(basedir=os.path.join(basedir, 'data'), skip_update_check=skip_update_check)
```

Comment:
Question: this does a path.join… should the rmtree call also do something similar?

Reply:
Makes sense to use an absolute path instead of a relative one